src/stuf/stuf_zds/client.py

```diff
@@ -114,13 +114,15 @@ def StufZDSClient(service: StufService, options: ZaakOptions) -> "Client":
         service,
         request_log_hook=partial(logevent.stuf_zds_request, service),
     )
-    return Client.configure_from(
+    client = Client.configure_from(
         factory,
         options=options,
         failure_log_callback=partial(logevent.stuf_zds_failure_response, service),
         success_log_callback=partial(logevent.stuf_zds_success_response, service),
         **init_kwargs,
     )
+    client.request = partial(client.request, timeout=service.soap_service.timeout)
+    return client
```

Comment:
ape-pie supports specifying `timeout` at the "client" level: https://github.com/maykinmedia/ape-pie/blob/979972db0797e0c0b50aa14db801c7e8e9ac170f/tests/test_config_adapter.py#L56

The timeout kwarg should be set in the `get_client_init_kwargs` helper.